upgrade_pythoncapi.py

```diff
@@ -578,15 +593,20 @@ def _parse_options(self, args):
         self.operations = self._get_operations(parser)
 
     def main(self):
-        for filename in self.walk(self.args.paths):
-            self.patch_file(filename)
+        if self.args.paths:
+            for filename in self.walk(self.args.paths):
+                self.patch_file(filename)
+
+        if self.args.download:
+            path = self.args.download
+            self.log(f"Download {PYTHONCAPI_COMPAT_H} to {path}")
+            self.get_latest_header(path)
 
         if self.pythoncapi_compat_added and not self.args.quiet:
             self.log()
             self.log(f"{INCLUDE_PYTHONCAPI_COMPAT} added: you may have "
                      f"to copy {PYTHONCAPI_COMPAT_H } to your project")
```

Comment:
minor unrelated suggestion:

```suggestion
                     f"to copy {PYTHONCAPI_COMPAT_H} to your project")
```